Pipfile

```diff
@@ -9,11 +9,10 @@ pylint = "*"
 [packages]
 Django = "==3.2.9"
 django-cors-headers = "*"
-django-oso = "*"
```

Comment:
What was the reason for removing this? I believe it is required to run the backend.